server/templates/staff/course/section/section.list.html

```diff
@@ -0,0 +1,104 @@
+<div class="box">
+    <div class="box-header">
+        <h3 class="box-title"><span> {{ title }} </span></h3>
+        <div class="box-tools">
+            <div class="box-tools pull-right">
+
+                <button type="button" class="btn btn-box-tool" data-widget="collapse" data-toggle="tooltip" title="Collapse">
+                    <i class="fa fa-minus"></i>
+                </button>
+
+            </div>
+        </div>
+    </div>
+    <div id="emails" style="display: none;">{{emails}}</div>
```

Comment:
This is rendered as:

<img width="205" alt="screen shot 2018-04-09 at 17 24 39" src="https://user-images.githubusercontent.com/2927722/38529649-13acd9ac-3c1b-11e8-89cd-236b6336719b.png">

It isn't clear that this is a intended to be clicked on or that it copies emails to the clipboard. It may be more clear if you use the bootstrap `btn` class. 

This renders as

<img width="190" alt="screen shot 2018-04-09 at 17 28 49" src="https://user-images.githubusercontent.com/2927722/38529721-8c32231e-3c1b-11e8-8356-63bdda3d0d77.png">